code/modules/mob/living/carbon/xenomorph/castes/Queen.dm

```diff
@@ -936,7 +934,7 @@
 	var/mob/living/carbon/xenomorph/queen/Queen = bound_xeno
 	if(Queen.ovipositor)
 		Queen.icon = Queen.queen_ovipositor_icon
-		Queen.icon_state = "[Queen.mutation_icon_state || Queen.mutation_type] Queen Ovipositor"
+		Queen.icon_state = "Normal Queen Ovipositor"
```

Comment:
Should we use the get strain here in case we ever have queen strains in future?

Reply:
Oh yeah good idea.